plugin/src/setKotlinMainActivity.ts

```diff
@@ -5,9 +5,18 @@ const breakingChangeVersion = 50
 const onActivityResultDeprecationVersion = 52 
 
 export function setKotlinMainActivity(contents: string, sdkVersion: number): string {
+  // imports
+  let imports = 
+  'import com.adyenreactnativesdk.AdyenCheckout\n' +
+  'import android.content.Intent\n';
+
+  if (sdkVersion >= onActivityResultDeprecationVersion) {
+   imports += 'import android.app.ComponentCaller\n';
```

Comment:
Should you treat this as an array of imports instead and check if another prebuild-plugin already has included them before including them?

Reply:
Because If you have f.x 4 imports of the same package you get:

```
e: file:///Users/username/repro/android/app/src/main/java/com/stefanwallin/repro/MainActivity.kt:6:20 Conflicting import: imported name 'ComponentCaller' is ambiguous.
e: file:///Users/username/repro/android/app/src/main/java/com/stefanwallin/repro/MainActivity.kt:7:20 Conflicting import: imported name 'ComponentCaller' is ambiguous.
e: file:///Users/username/repro/android/app/src/main/java/com/stefanwallin/repro/MainActivity.kt:8:20 Conflicting import: imported name 'ComponentCaller' is ambiguous.
e: file:///Users/username/repro/android/app/src/main/java/com/stefanwallin/repro/MainActivity.kt:9:20 Conflicting import: imported name 'ComponentCaller' is ambiguous.

Reply:
Good point!

Reply:
Thanks @StefanWallin 

Addressed in [2d94c71](https://github.com/Adyen/adyen-react-native/pull/607/commits/2d94c7159ae72187090b76f695b9b9d0555398e4)